src/helpers.php

```diff
@@ -96,3 +96,15 @@ function dbdecrypt(string $value): ?string
         return database_decrypt($value);
     }
 }
+
+if (! function_exists('str_random')) {
+
+    /**
+     * @param int $value
+     * @return string|null
+     */
+    function str_random(int $value = 16): ?string
```

Comment:
Why? Just replace it with `Illuminate\Support\Str::random()` if it's even being used anywhere.